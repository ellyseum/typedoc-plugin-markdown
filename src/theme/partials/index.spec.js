const context = {
  groups: [
    {
      children: [
        {
          name: 'amount',
          url: 'modules/_variables_.md#amount',
        },
        {
          name: 'color',
          url: 'modules/_variables_.md#color',
        },
        {
          name: 'isDone',
          url: 'modules/_variables_.md#isdone',
        },
        {
          name: 'numbers',
          url: 'modules/_variables_.md#numbers',
        },
      ],
      title: 'Variables',
      someChildrenAreExported: true,
      allChildrenAreInherited: false,
      allChildrenArePrivate: false,
      allChildrenAreProtectedOrPrivate: false,
      allChildrenAreExternal: false,
    },
  ],
  args: {},
};

describe(`index.hbs`, () => {
  test('should compile', () => {
    const result = global.compileTemplate('index.hbs', context);
    expect(result).toMatchSnapshot();
  });
});
