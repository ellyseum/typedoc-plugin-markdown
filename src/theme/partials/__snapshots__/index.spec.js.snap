// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.hbs should compile 1`] = `
"## Index

### Variables

* [amount]([relativeURL])
* [color]([relativeURL])
* [isDone]([relativeURL])
* [numbers]([relativeURL])"
`;
